aerleon/lib/openconfig.py

```diff
@@ -24,7 +24,7 @@
 import json
 import sys
 from collections import defaultdict
-from typing import Any, DefaultDict, Dict, List, Set, Tuple, Union
```

Comment:
You probably want to continue to use Dict and List. Any reason you decided to remove that here?

Reply:
Oops, my bad, looks like they're built-in only starting Python 3.9? Anyway, reverted.